ui-v2/tests/acceptance/dc/nspaces/index.feature

```diff
@@ -10,7 +10,15 @@ Feature: dc / nspaces / index: Nspaces List
       Namespace: default
     ---
     And 1 datacenter model with the value "dc-1"
-    And 3 nspace models
+    And 3 nspace models from yaml
+    ---
+    - Name:  a-namespace
+      Description: a namespace
+    - Name:  default
+      Description: The default namespace
+    - Name:  z-namespace
+      Description: z namespace
+    ---
```

Comment:
This isn't really needed, and it just replicates what comes out of the 'default mocks', but there is a tiny chance that at some point in the future if we were to change anything in the default mocks here, we may not get the same amount of namespaces before and after the default one, as we explicitly setup some mocks here, that means we can guarantee we'll always get the same mock data.